modules/core/src/main/java/org/apache/ignite/internal/processors/cache/ClusterCachesInfo.java

```diff
@@ -127,6 +127,8 @@
     /** Dynamic caches. */
     private final ConcurrentMap<String, DynamicCacheDescriptor> registeredCaches = new ConcurrentHashMap<>();
 
+    private final ConcurrentMap<Integer, DynamicCacheDescriptor> cacheIdToNameMapping = new ConcurrentHashMap<>();
```

Comment:
Please change the field name according to the new type (something like registeredCachesById, to match naming of other fields). Don't forget to rename methods and fix the method's javadoc.
Also, javadoc is required for all fields by codestyle (at least `/** */`) see https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines#CodingGuidelines-JavadocComments